### ISSUE TITLE
Document the configuration file and options for customizing doT parser

### ISSUE THREAD

**Issue by rameshdavid** [lorefnon/InGenR#4]
Looking at the [code here](https://github.com/lorefnon/InGenR/blob/57b5676ea3afa0a7ce53af7cc731438cf9f9a7a3/src/commands.ts#L20) it seems like InGenR uses cosmiconfig to support a configuration file, however this is not documented at all. 

I think this is particularly important because by using the config file we can change the dot template delimiters for languages where the default delimiters are problematic (the templates need a lot of escaping).

**Comment by lorefnon** [comment]
Yes, this project is very new and not many features are well documented. 

If you'd like to contribute a PR for this I'd really appreciate it.
